assets/aml-benchmark/tests/pipelines/batch_benchmark_score.yaml

```diff
@@ -0,0 +1,42 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+
+display_name: test_endpoint_score
+description: Pipeline to test endpoint score.
+settings:
+  default_compute: azureml:serverless
+
+inputs:
+  data_input_table:
+    type: uri_folder
+    path: ../data/
+  online_endpoint_url: 'https://yuzhua-ws-eastus-mxawd.eastus.inference.ml.azure.com/score'
```

Comment:
I saw in some tests you deploy an endpoint. Is this URL used anywhere?

Reply:
removed